README.md

```diff
@@ -1,3 +1,21 @@
-#SUPERDESK Content API
+# Superdesk Content API
 [![Build Status](https://travis-ci.org/superdesk/superdesk-content-api.svg?branch=master)](https://travis-ci.org/superdesk/superdesk-content-api)
-Superdesk Content API provides a well defined contract specifying how to consume content produced by Superdesk.
+
+In today's fast-changing media landscape, digital news content is not longer
+consumed in the Web only, but in a diversity of platforms, services and
+devices. The Superdesk Content API allows news organizations and content
+publishers using [Superdesk](https://github.com/superdesk/superdesk) to make
+their content available for delivering everywhere.
+
+## Overview
+
+The Superdesk Content API is a read-only API focused on delivering news content;
+news items and packages created, produced and published in Superdesk.
+
+The Superdesk Content API is an open source project developed and maintained by
+[Sourcefabric](https://www.sourcefabric.org).
+
+## Documentation
+
+Check it out at http://docs.superdeskcontentapi.apiary.io/
```

Comment:
just wondering - it could also use label so maybe: [apiary](http://docs.superdeskcontentapi.apiary.io/) ?